src/tests/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 find_package(Boost 1.39.0 REQUIRED COMPONENTS unit_test_framework)
-foreach(PROG test_elements )
+foreach(PROG test_elements test_edge )
```

Comment:
Why not use `file(GLOB test_cc test_*.cc)` to get the list of tests; you can use `get_filename_component(fname ${PROG} NAME_WE)` to get the file name without the extension of the directory.

https://cmake.org/cmake/help/v3.0/command/get_filename_component.html

Reply:
That seems better, I'm just not yet that familiar with cmake will definitely implement.

Reply:
I would prefer listing the tests explicitly in case we have to disable a test.

I would use something like
```cmake
foreach(TEST elements edge)
```
though.

Reply:
Ok I will leave it then :)